### PR TITLE
DBZ-7947 Check for read-only mode incompatibility

### PR DESCRIPTION
**Pull request by mfvitale**
closes: https://issues.redhat.com/browse/DBZ-7947

### CODE REVIEW

**Comment by mfvitale** [comment]
This failure are fixed with https://github.com/debezium/debezium/pull/5624/files#diff-262dfe4690606ccae9bae002aea572b8750007242d5a7628efb8052c69dff047R206. 

It is related to the recent change on the PostgreSQL version used in the tests (10 -> 12).

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java [hunk 1]
```diff
@@ -89,6 +91,19 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
         final Charset databaseCharset;
         try (PostgresConnection tempConnection = new PostgresConnection(connectorConfig.getJdbcConfig(), PostgresConnection.CONNECTION_GENERAL)) {
             databaseCharset = tempConnection.getDatabaseCharset();
+
```
Comment (jpechane) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java:94]: @mfvitale Should not this check go to `io.debezium.connector.postgresql.PostgresConnector.validateConnection(Map<String, ConfigValue>, Configuration)`?

Comment (mfvitale) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java]: Definitely a better place. Thanks.

**Comment by jpechane** [comment]
@mfvitale Applied, thanks!

**Pull request merged**